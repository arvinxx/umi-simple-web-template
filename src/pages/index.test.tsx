import { render } from '@testing-library/react';

import App from './index';

describe('App', () => {
  it('默认', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
