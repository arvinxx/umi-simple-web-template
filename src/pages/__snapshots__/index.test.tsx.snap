// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 默认 1`] = `
.emotion-0 {
  font-family: Georgia,sans-serif;
  margin-top: 4em;
  text-align: center;
}

.emotion-1 {
  height: 512px;
  background: url() no-repeat center 0;
  -webkit-background-size: 512px 512px;
  background-size: 512px 512px;
}

.emotion-2 {
  font-size: 1.2em;
  margin: 1.8em 0 0;
  list-style: none;
  line-height: 1.5em;
}

.emotion-3 {
  padding: 8px;
  background: #f5f5f5;
}

<div>
  <div
    class="emotion-0"
  >
    <div
      class="emotion-1"
    />
    <ul
      class="emotion-2"
    >
      <li
        class="emotion-3"
      >
        To get started, edit 
        <code>
          src/pages/index.js
        </code>
         and save to reload.
      </li>
      <li
        class="emotion-3"
      >
        <a
          href="https://umijs.org/guide/getting-started.html"
        >
          Getting Started
        </a>
      </li>
    </ul>
  </div>
</div>
`;
